pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -119,21 +176,21 @@ class PillarboxCastPlayer(
     }
 
     override fun addListener(listener: Player.Listener) {
-        castPlayer.addListener(listener)
+        castPlayer.addListener(ForwardingListener(this, listener))
         listeners.add(listener)
     }
 
+    @SuppressLint("ImplicitSamInstance")
     override fun removeListener(listener: Player.Listener) {
-        castPlayer.removeListener(listener)
+        castPlayer.removeListener(ForwardingListener(this, listener))
         listeners.remove(listener)
     }
 
     override fun getAvailableCommands(): Player.Commands {
-        val isShuffleAvailable = remoteMediaClient?.mediaStatus?.isMediaCommandSupported(MediaStatus.COMMAND_QUEUE_SHUFFLE) == true
```

Comment:
Why remove this?

Reply:
Because I failed to use git :P Well done by seeing that.